helm/spark-yotpo/templates/sparkapplication.yaml

```diff
@@ -0,0 +1,161 @@
+apiVersion: sparkoperator.k8s.io/v1beta2
+kind: SparkApplication
+metadata:
+  name:  {{ .Release.Name | trunc 63 }}
```

Comment:
why the trunc?
The realeas name will be different on every use deploy.
I would guess the name here would be something like:
"loaded-galeries"